### PR TITLE
Fix tests with bx auth

### PR DESCRIPTION
**Pull request by AnthonyAmanse**
Replaced `BLUEMIX_AUTH` to `TRAVIS_PULL_REQUEST`

Quick change of `$NODE_PORT` variable to `30080`

### CODE REVIEW

File: scripts/bx_auth.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 BLUEMIX_ORG="Developer Advocacy"
 BLUEMIX_SPACE="dev"
 
-if [[ -z "$BLUEMIX_AUTH" ]]; then
+if [[ -z "$BLUEMIX_API_KEY" ]]; then
```
Comment (mlangbehn) [inline scripts/bx_auth.sh]: Quick point, this isn't actually testing if we are in a fork, it is only testing if `BLUEMIX_API_KEY` is set. Would it be better to check if [`TRAVIS_PULL_REQUEST`](https://docs.travis-ci.com/user/environment-variables/#Convenience-Variables) is set (see [here](https://github.com/IBM/Kubernetes-container-service-GitLab-sample/blob/master/scripts/bx_auth.sh#L6) for example)?

File: scripts/install_bx.sh [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e
 
-if [[ -z "$BLUEMIX_AUTH" ]]; then
+if [[ -z "$BLUEMIX_API_KEY" ]]; then
```
Comment (mlangbehn) [inline scripts/install_bx.sh]: See comment from scripts/bx_auth.sh.

File: tests/test-kubeadm-dind-cluster.sh [hunk 3]
```diff
@@ -70,7 +70,7 @@ verify_deploy(){
 
 main(){
 
-    if [[ -z "$BLUEMIX_AUTH" ]]
+    if [[ -z "$BLUEMIX_API_KEY" ]]
```
Comment (mlangbehn) [inline tests/test-kubeadm-dind-cluster.sh]: See comment from scripts/bx_auth.sh.

File: tests/test-kubernetes.sh [hunk 4]
```diff
@@ -91,7 +91,7 @@ verify_deploy(){
 }
 
 main(){
-    if [[ -z "$BLUEMIX_AUTH" ]]; then
+    if [[ -z "$BLUEMIX_API_KEY" ]]; then
```
Comment (mlangbehn) [inline tests/test-kubernetes.sh]: See comment from scripts/bx_auth.sh.

**Comment by AnthonyAmanse** [comment]
@mlangbehn `$BLUEMIX_API_KEY` changed to `$TRAVIS_PULL_REQUEST`

**Comment by AnthonyAmanse** [comment]
@mlangbehn `BLUEMIX_API_KEY` changed to check for `TRAVIS_PULL_REQUEST`

**Pull request closed without merging**